scripts/fast-reboot-dump.py

```diff
@@ -39,7 +39,7 @@ def generate_neighbor_entries(filename, all_available_macs):
         }
         arp_output.append(obj)
 
-        ip_addr = key.split(':')[2]
+        ip_addr = key.split(':', 2)[2]
```

Comment:
Thanks for this fix. I believe below 3 lines should be removed now.

Reply:
This change fixes the issue as follows:
```
Feb  2 02:33:29.360356 cab18-3-dut ERR fast-reboot-dump: Got an exception 'fc02' does not appear to be an IPv4 or IPv6 interface: Traceback: Traceback (most recent call last):
#012  File "/usr/local/bin/fast-reboot-dump.py", line 301, in <module>
#012    res = main()#012  File "/usr/local/bin/fast-reboot-dump.py", line 292, in main
#012    neighbor_entries = generate_neighbor_entries(root_dir + '/arp.json', all_available_macs)
#012  File "/usr/local/bin/fast-reboot-dump.py", line 43, in generate_neighbor_entries
#012    if ipaddress.ip_interface(str(ip_addr)).ip.version != 4:
#012  File "/usr/lib/python3.7/ipaddress.py", line 119, in ip_interface
#012    address)
#012ValueError: 'fc02' does not appear to be an IPv4 or IPv6 interface
Feb  2 02:33:29.371251 cab18-3-dut NOTICE admin: fast-reboot failure (12) cleanup ...
```

Reply:
Same issue exists in 202012 branch. Please cherry-pick.

Reply:
@akokhan if this fixes the issue you mention, why you think we need to remove these lines?

Reply:
From what I see, the following code just prefix from `NEIGH_TABLE:Vlan1000:200:200:200:200::1` to get IPv6 address. The line that you've fixed, does the same. Correct? So, we should be good to remove those `if` block. Please double check.
```
        if ipaddress.ip_interface(str(ip_addr)).ip.version != 4:
            #This is ipv6 address
            ip_addr = key.replace(key.split(':')[0] + ':' + key.split(':')[1] + ':', '')
```

Reply:
Now I get it, yes I agree, will double check it
thanks